include/solvers/heat_transfer.h

```diff
@@ -186,110 +222,160 @@ class HeatTransfer : public AuxiliaryPhysics<dim, GlobalVectorType>
 
 
   /**
-   * @brief pre_mesh_adaption Prepares the auxiliary physics variables for a
-   * mesh refinement/coarsening
+   * @brief Prepare the auxiliary physics variables for a
+   * mesh refinement/coarsening.
    */
   void
   pre_mesh_adaptation() override;
 
   /**
-   * @brief post_mesh_adaption Interpolates the auxiliary physics variables to the new mesh
+   * @brief Interpolate the auxiliary physics variables to the new mesh.
    */
   void
   post_mesh_adaptation() override;
 
   /**
    * @brief Compute the Kelly error estimator for mesh refinement.
    *
-   * @param ivar The current element of the map simulation_parameters.mesh_adaptation.variables
+   * @param ivar The current element of the map simulation_parameters.mesh_adaptation.variables.
    *
-   * @param estimated_error_per_cell The deal.II vector of estimated_error_per_cell
+   * @param estimated_error_per_cell The deal.II vector of estimated_error_per_cell.
    */
   void
   compute_kelly(const std::pair<const Parameters::MeshAdaptation::Variable,
                                 Parameters::MultipleAdaptationParameters> &ivar,
                 dealii::Vector<float> &estimated_error_per_cell) override;
 
   /**
-   * @brief Prepares Heat Transfer to write checkpoint
+   * @brief Prepare Heat Transfer to write checkpoint
    */
   void
   write_checkpoint() override;
 
   /**
-   * @brief Allows Heat Transfer to set-up solution vector from checkpoint file;
+   * @brief Allow Heat Transfer to set-up solution vector from checkpoint file.
    */
   void
   read_checkpoint() override;
 
   /**
-   * @brief Sets-up the DofHandler and the degree of freedom associated with the physics.
+   * @brief Set-up the DofHandler and the degree of freedom associated with the physics.
    */
   void
   setup_dofs() override;
 
   /**
-   * @brief Sets-up the initial conditions associated with the physics. Generally, physics
-   * only support imposing nodal values, but some physics additionnaly support
-   * the use of L2 projection or steady-state solutions.
+   * @brief Set-up the initial conditions associated with the physics.
+   * Generally, physics only support imposing nodal values, but some physics
```

Comment:
I would put this comment in Auxiliary Physics and write specifically what Heat Transfer allows as initial conditions.